src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,38 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class Manager extends AbstractEmployee {
+
+    private List<Employee> myEmployees;
+
+    public Manager(String name, double salary) {
+        super(name, salary);
+        this.myEmployees = new ArrayList<>();
+    }
+
+    public int howManyEmployees() {
+        return myEmployees.size();
+    }
+
+    public Employee getEmployee(int i) {
+        return myEmployees.get(i);
+    }
+
+    public void hireEmployee(Employee newCandidate) {
+        myEmployees.add(newCandidate);
+    }
+
+    public double myEmployeesSalaries() {
+        double sum = 0;
+        int count = myEmployees.size();
+        for (int i = 0; i < count; i++) {
```

Comment:
Pewnie przyzwyczajenie z Cpp :D
W javie chwalimy i używamy pętli foreach (nawet Intellij podpowiada jej użycie - podświetla `for` na żółto. Można nakliknąć tam kursorem, alt + 2x enter i nam zamieni)
`for (Employee myEmployee : myEmployees) {
            sum += myEmployee.getSalary();
        }`
Taka pętla wygląda ładniej, można powiedzieć - bardziej wysokopoziomowo :)